macos/FluentUI/Notification/Notification.swift

```diff
@@ -0,0 +1,346 @@
+                messageButtonAction: (() -> Void)? = nil) {
+        let state = MSFNotificationStateImpl(style: style,
+                                             message: message,
+                                             title: title,
+                                             image: image,
+                                             actionButtonTitle: actionButtonTitle,
+                                             actionButtonAction: actionButtonAction,
+                                             messageButtonAction: messageButtonAction)
+		self.state = state
+		self.backgroundColor = {
+			switch state.style {
+			case .primaryToast, .primaryBar:
+				return Constants.primaryBackgroundColor
+			case .neutralToast, .neutralBar:
+				return Constants.neutralBackgroundColor
+			case .primaryOutlineBar:
+				return Constants.primaryOutlineBarBackgroundColor
+			case .dangerToast:
+				return Constants.dangerToastBackgroundColor
+			case .warningToast:
+				return Constants.warningToastBackgroundColor
+			}
+		}()
+		self.foregroundColor = {
+			switch state.style {
+			case .primaryToast, .primaryBar:
+				return Constants.primaryForegroundColor
+			case .neutralToast, .neutralBar:
+				return Constants.neutralForegroundColor
+			case .primaryOutlineBar:
+				return Constants.primaryOutlineBarForegroundColor
+			case .dangerToast:
+				return Constants.dangerToastForegroundColor
+			case .warningToast:
+				return Constants.warningToastForegroundColor
+			}
+		}()
+	}
+
+	public var body: some View {
+		@ViewBuilder
+		var image: some View {
+			if state.style.isToast {
+				if let image = state.image {
+					let imageSize = image.size
+					Image(nsImage: image)
+						.frame(width: imageSize.width, height: imageSize.height, alignment: .center)
+						.foregroundColor(foregroundColor)
+				}
+			}
+		}
+
+		@ViewBuilder
+		var titleLabel: some View {
+			if state.style.isToast && hasSecondTextRow {
+				if let title = state.title {
+					Text(title)
+						.font(.system(size: 15))
+						.fontWeight(.semibold)
+						.foregroundColor(foregroundColor)
+						.fixedSize(horizontal: false, vertical: true)
+				}
+			}
+		}
+
+		@ViewBuilder
+		var messageLabel: some View {
+			if let message = state.message {
+				Text(message)
+					.font(.system(size: 15))
+					.fontWeight(.regular)
+					.foregroundColor(foregroundColor)
+					.fixedSize(horizontal: false, vertical: true)
+			}
+		}
+
+		@ViewBuilder
+		var textContainer: some View {
+			VStack(alignment: .leading) {
+				if hasSecondTextRow {
+					titleLabel
+				}
+				messageLabel
+			}
+			.padding(.vertical, Constants.verticalPadding)
+		}
+
+		@ViewBuilder
+		var button: some View {
+			if let buttonAction = state.actionButtonAction ?? nil {
+				if let actionTitle = state.actionButtonTitle, !actionTitle.isEmpty {
+					SwiftUI.Button(action: {
+						buttonAction()
+					}, label: {
+						if #available(macOS 13.0, *) {
```

Comment:
fluent framework supports only macOS15+ so you shouldn't need this

Reply:
The readme, as well as `FluentUI_common.xcconfig` still have macOS 10.15, should they be updated?

Reply:
clearly i am dumb. and there is difference between 10.15 vs macos13